File: src/nl/rubensten/texifyidea/action/insert/InsertPartAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertPartAction extends InsertEditorAction {

    public InsertPartAction() {
        super("Part", TexifyIcons.DOT_PART, "\\part{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertPartAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertPartAction extends InsertEditorAction {

    public InsertPartAction() {
        super("Part", TexifyIcons.DOT_PART, "\\part{", "}");
    }
}
